kura/distrib/pom.xml

```diff
@@ -1066,10 +1066,10 @@
 
     <profiles>
         <profile>
-            <id>raspberry-pi-3-4-ubuntu-20</id>
+            <id>raspberry-ubuntu-20</id>
```

Comment:
I think the profile name should be `raspberry-pi-ubuntu-20` right?

Reply:
yeah, i lost a pi!